jssource/minify_utils.php

```diff
@@ -504,3 +504,102 @@ function BackUpAndCompressScriptFiles($from_path,$to_path = '', $backup = true){
             }
 
         }
+
+
+/**
+ * Join and minify JS files
+ *
+ * @param array $jsFiles an 'array' of js files
+ *
+ * @author Jose C. Massón <jose@gcoop.coop>
+ *
+ * @return Minified JS file path
+ */
+function joinAndMinifyJSFiles($jsFiles)
+{
+    $jsFiles = array_map('realpath', array_unique($jsFiles));
```

Comment:
More verbose if you do something like this:
```
$jsFilesReal = [];
foreach ($jsFiles as $jsFilesKey => $jsFile) {
  $jsFileReal = realpath($jsFile);
  if (!$jsFileReal) {
    LoggerManager::getLogger()->error('joinAndMinifyJSFiles - Given JS File path is wrong: ' . $jsFile);
  } else {
    if (in_array($jsFileReal, $jsFilesReal)) {
      LoggerManager::getLogger()->warn('joinAndMinifyJSFiles - Given JS File is duplicated: ' . $jsFile);
    } else {
      $jsFilesReal[$jsFilesKey] = $jsFileReal;
    }
  }
}
$jsFiles = $jsFilesReal;
```

Reply:
Done